canonical-web-and-design/ubuntu.com
# Display the renewal settings

## Done

- Fetch user info from the API.
- Display the renewal settings data.

## QA

- Visit [/advantage?test_backend=true](https://ubuntu-com-10387.demos.haus/advantage?test_backend=true) and log in.
- Click on "Renewal settings" at the top of the list.
- The number of subs, price and renewal date are displayed from the API data.
- The 'Auto-renewal' checkbox should reflect the state of your auto renewal settings (probably on).
- Note: the form doesn't update your settings yet (will be done in https://github.com/canonical-web-and-design/commercial-squad/issues/272).

## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/99.

## Screenshots


<img width="455" alt="Screen Shot 2021-09-17 at 1 16 13 pm" src="https://user-images.githubusercontent.com/361637/133718819-ea2e05d9-ff96-4aa3-93f8-fc821c258d1b.png">
